src/test/java/seedu/address/logic/commands/doctor/ClearDoctorCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands.doctor;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalAppObjects.getTypicalDoctorRecords;
+import static seedu.address.testutil.TypicalAppObjects.getTypicalPatientRecords;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.AppointmentSchedule;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ClearDoctorCommandTest {
+
```

Comment:
note: to modify test classes here to test correctly after delete-patient/delete-doctor bugfix has been implemented